hugolib/menu.go

```diff
@@ -25,9 +25,11 @@ import (
 // or in the site config.
 type MenuEntry struct {
 	URL        string
+	Page       *Page // probably only set when called from Page front matter
```

Comment:
Remove this comment. The "probably" makes me start to think that we may have missed something, which I don't think, so this will just confuse the next person looking at this code.

Reply:
fixed